examples/preprocessing/plot_css.py

```diff
@@ -0,0 +1,96 @@
+"""
+=================================================================
+Cortical Signal Suppression (CSS) for removal of cortical signals
+=================================================================
+
+This script shows an easy example of how to use CSS. CSS
+suppresses the cortical contribution to the signal subspace
+in EEG data using MEG data, facilitating detection of subcortical
+signals. We will illustrate how it works by simulating one cortical
+and one subcortical oscillation at different frequencies; 40 Hz and
+239 Hz for cortical and subcortical activity, respectively, then
+process it with CSS and look at the power spectral density of the
+raw and processed data.
+
+References
+----------
+.. [1] Samuelsson J, Khan S, Sundaram P, Peled N, Hamalainen M (2019)
+Cortical Signal Suppression (CSS) for detection of subcortical activity using
+MEG and EEG, Brain topography 32 (2), 215-228.
```

Comment:
this is not anymore how we cite papers. We have a proper bibtex file now.

See how it's done elsewhere

Reply:
Couldn't find any mentioning of this in the contribution guide or elsewhere, but added the citation to doc/references.bib and removed this from the file, I hope this is the right way.

Reply:
Here is an example of the new way of handling citations:

https://github.com/mne-tools/mne-python/blob/92a41a284093a845697b81dab89afed8bc3eeb4b/mne/preprocessing/ica.py#L185

...and here is the corresponding line at the end of the docstring:

https://github.com/mne-tools/mne-python/blob/92a41a284093a845697b81dab89afed8bc3eeb4b/mne/preprocessing/ica.py#L347-L349

For examples/tutorials it's basically the same:

https://github.com/mne-tools/mne-python/blob/92a41a284093a845697b81dab89afed8bc3eeb4b/tutorials/preprocessing/plot_60_maxwell_filtering_sss.py#L36

followed by

https://github.com/mne-tools/mne-python/blob/92a41a284093a845697b81dab89afed8bc3eeb4b/tutorials/preprocessing/plot_60_maxwell_filtering_sss.py#L340-L343

I'll open a PR to add this guidance to the contributors guide; thanks for pointing out that it was missing.

Reply:
Great, thanks! I added the footcites as suggested, hopefully it's right now.